### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tinytb**
Minor edits, but hopefully gets the README "up to date" in terms of the current state

### CODE REVIEW

**Review by 2color — APPROVED** [review]
Looks better!

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
-<h3 align="center">An implementation of IPFS in JavaScript.</h3>
+# Helia — an implementation of IPFS in JavaScript
 
 ### Project status
 
-This project is pre-alpha and is planned for development in late 2022, with an initial "v1" release in [late Q1 2023](/ROADMAP.md#late-q1-march). The project will be built in the open. Community contributors are welcome.
+This project is pre-alpha and is currently in development. An initial v1 release is planned for [late Q1 2023](/ROADMAP.md#late-q1-march). Helia is being built in the open; community contributors are welcome!
+
+The core of IPFS is the [Files API](https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/FILES.md), which will likewise be implemented in Helia. These initial building blocks are in development now; have a look at this repo's PR(s). For more info about Helia, please see the [Roadmap](https://github.com/ipfs/helia/issues/5) and the [Manifesto](MANIFESTO.md).
```
Comment (SgtPooki) [inline README.md:7]: One roadmap links to ipfs/helia#5 and one links to ./ROADMAP.md. Should we consolidate them?

Comment (SgtPooki) [inline README.md:7]: One roadmap links to ipfs/helia#5 and one links to ./ROADMAP.md. Should we consolidate them?

File: README.md [hunk 2]
```diff
@@ -27,11 +31,11 @@ For more information, see the [State of IPFS in JS (blog post)](https://blog.ipf
 
 ## Roadmap
 
-The roadmap can be found here: https://github.com/ipfs/pomegranate/issues/5
+Please find and comment on [the Roadmap here](https://github.com/ipfs/pomegranate/issues/5).
```
Comment (SgtPooki) [inline README.md:34]:
```suggestion
Please find and comment on [the Roadmap here](https://github.com/ipfs/helia/issues/5).
```

**Review by SgtPooki — APPROVED** [review]
approved with suggestions for improvement.

File: README.md (hunk 1 above)
Comment (tinytb) [reply README.md:7]: #5 is where readers can leave comments. #5 links to ROADMAP.md, so I suppose #5 should be the place to go.

**Pull request merged**